### PR TITLE
Login React

### PR DESCRIPTION
**Pull request by Abdelrahman-Rabi**
user can login and get the JWT saved in localstorage 

image

### CODE REVIEW

File: front-end/src/components/Login.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+import axios from "axios";
+
+const Login = () => {
+  const [values, setValues] = useState({
+    email: "",
+    password: "",
+  });
+
+  const handleChange = (e) => {
+    console.log(e);
```
Comment (Besslan) [inline front-end/src/components/Login.js]: Remove the console log

File: front-end/src/components/Login.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    if (!values.password) {
+      errors.password = "Password Required";
+    } else if (values.password.length < 8) {
+      errors.password = "Incorrect Password";
+    }
+    return errors;
+  };
+
+  const handleSubmit = (e) => {
+    console.log("Values:", values);
```
Comment (Besslan) [inline front-end/src/components/Login.js]: remove the console log

File: front-end/src/components/Login.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+        // Bath from BE
+        .post("http://localhost:5000/login", values)
+        .then((result) => {
+          console.log("Result: ", result);
+          if (!result.data.error) {
+            localStorage.setItem("token", result.data);
+          }
+          //   else {
+          //     setErrors({ ...errors, validation: "Invalid Email or Password" });
+          //   }
```
Comment (Besslan) [inline front-end/src/components/Login.js]: remove the commented out code

File: front-end/src/components/Login.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+    setErrors(validate(values));
+    console.log("errors:", errors);
+    if (Object.keys(errors).length) {
+      setErrors({ ...errors, validation: "Invalid Email or Password" });
+    } else {
+      axios
+        // Bath from BE
+        .post("http://localhost:5000/login", values)
+        .then((result) => {
+          console.log("Result: ", result);
```
Comment (Besslan) [inline front-end/src/components/Login.js]: remove the console log

File: front-end/src/components/Login.js [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        <label>Password </label>
+        <br />
+        <input
+          type="password"
+          name="password"
+          placeholder="Enter your Password"
+          value={values.password}
+          onChange={handleChange}
+        ></input>
+        {/* {errors.password && <p> {errors.password} </p>} */}
```
Comment (Besslan) [inline front-end/src/components/Login.js]: remove the commented out line

File: front-end/src/components/Login.js (hunk 1 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Login.js]: Done

File: front-end/src/components/Login.js (hunk 2 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Login.js]: Done

File: front-end/src/components/Login.js (hunk 3 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Login.js]: Done

File: front-end/src/components/Login.js (hunk 4 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Login.js]: Done

File: front-end/src/components/Login.js (hunk 5 above)
Comment (Abdelrahman-Rabi) [reply front-end/src/components/Login.js]: Done

**Pull request merged**